frontend/app/src/composables/balances/refresh.ts

```diff
@@ -36,12 +35,13 @@ export const useRefresh = createSharedComposable(() => {
     const chains = chain ? arrayify(chain) : get(txEvmChains).map(chain => chain.id);
 
     set(massDetecting, chain || 'all');
-    await awaitParallelExecution(
-      chains,
-      chain => chain,
-      async chain => useTokenDetection(chain).detectTokensOfAllAddresses(),
-      1,
+
+    // Call all detections - they're queued internally anyway
+    const detectionPromises = chains.map(async chain =>
+      useTokenDetection(chain).detectTokensOfAllAddresses(),
```

Comment:
what does this do internally? 
Queue the address detection for all chains for all addresses?